Sources/Lumina/UI/LuminaViewController.swift

```diff
@@ -363,6 +347,10 @@ open class LuminaViewController: UIViewController {
       }
     }
   }
+  
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/dokun1/Lumina/blob/fad22d081b489a17a78d68bf07f146bf4c46f417/Sources/Lumina/UI/LuminaViewController.swift#L350"></span>Lines should not have trailing whitespace.<br />
<code>trailing_whitespace</code> <code>LuminaViewController.swift:350</code></td>
    </tr>
  </tbody>
</table>